pkg/util/docker/global.go

```diff
@@ -67,8 +67,7 @@ func EnableTestingMode() {
 	})
 }
 
-// HostnameProvider docker implementation for the hostname provider
-func HostnameProvider(ctx context.Context) (string, error) {
+func HostnameProvider(ctx context.Context, options map[string]interface{}) (string, error) {
```

Comment:
Exported function should be commented.

Reply:
Hmm - I don't know where this disappeared! 👍